.github/workflows/ci.yml

```diff
@@ -2,9 +2,14 @@ name: "Terraform CKA Lab VMs"
 
 on:
   push:
-    branches: ["*"]
+    branches: [main]
     paths:
-      - 'proxmox/terraform/**'
```

Comment:
I intentionally set to terraform folder as for the other terraform-init folder it is one-off run and should not be run in GH action

Reply:
no, this workflow only run for this specific terraform location